### PR TITLE
Add test DAG 

### PR DESCRIPTION
**Pull request by gheise**
Add test DAG we're using to test database connections
---
We're using a basic DAG to test creating new connections and addressing the infra required to access the databases through airflow, adding it here for transparency and maintenance

Related issue:

- #1

### CODE REVIEW

File: test_postgres.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import datetime
+import os
+
+from airflow import DAG
+from airflow.providers.postgres.operators.postgres import PostgresOperator
```
Comment (come-maiz) [inline test_postgres.py:5]: on python, I prefer to import at the module level.
`import airflow`

Then use `airflow.DAG`.

This gives a lot of context when reading the code, about which module you are using things from.

File: test_postgres.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+DAG_ID = "mev_db_test_dag"
```
Comment (come-maiz) [inline test_postgres.py:7]: too many acronyms. At least database can be expanded.
or what about `mev_database_test_graph` ?

File: test_postgres.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+with DAG(
+    dag_id = DAG_ID,
+    start_date = datetime.datetime(2022,9,6),
+    schedule_interval = "@once",
+    catchup = False,
+) as dag:
+    get_count = PostgresOperator(
+        postgres_conn_id="mev_db",
```
Comment (come-maiz) [inline test_postgres.py:16]:
```suggestion
        postgres_connection_id="mev_db",
```

**Review by come-maiz — COMMENTED** [review]
Shouldn't this be in a tests directory?

File: test_postgres.py (hunk 1 above)
Comment (gheise) [reply test_postgres.py:5]: I usually seek for guidance from the official guidelines. According to the Python style guidelines, the import be at the last level. You can read up on it [here](https://peps.python.org/pep-0328/)

File: test_postgres.py (hunk 3 above)
Comment (gheise) [reply test_postgres.py:16]: this is a paramater, which can't be changed. ```postgres_conn_id``` is the airflow-defined param

**Review by taarushv — APPROVED** [review]

File: test_postgres.py (hunk 3 above)
Comment (come-maiz) [reply test_postgres.py]: :disappointed-in-humanity:

File: test_postgres.py (hunk 1 above)
Comment (come-maiz) [reply test_postgres.py]: I've read it and I couldn't find where it says anything about importing at the last level. 

As long as it's consistent, I'm happy. I can keep my preference for module import vrs class imports for a project I maintain.

**Pull request merged**